org.civicrm.reqangular/src/common/services/pub-sub.js

```diff
@@ -0,0 +1,59 @@
+define([
+  'common/modules/services'
+], function (services) {
+  'use strict';
+
+  var topics = Object.create(null);
+
+  /**
+   * This service is used for communication between different angular apps using pub-sub pattern
+   * @ngdoc service
+   * @name pubSub
+   * @requires $rootScope
+   **/
```

Comment:
Sorry, still find syntax and styling problems with your documentation, let's be on the same page about it:
```js
/**
 * This service is used for communication between different angular apps using pub-sub pattern
 *
 * @ngdoc service
 * @name pubSub
 * @requires $rootScope
 */
services.factory('pubSub', /* .. */)
```